<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ISAM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequential</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>541</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 213, 0); 
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="target1">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string notr="true">name to search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QpushButton{
	background-color: #A9A9A9;
	border: 3px solid #696969;
}
QpushButton:pressed{
	background-color: #FFFAF0;
	border: 3px solid #696969;
}</string>
         </property>
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="target2">
         <property name="placeholderText">
          <string>name,user,mail,pass</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">QpushButton{
	background-color: #A9A9A9;
	border: 3px solid #696969;
}
QpushButton:pressed{
	background-color: #FFFAF0;
	border: 3px solid #696969;
}</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="target3">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>name to delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Accesses">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetResult">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Result: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Timer">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Message">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
